googlechat/googlechat.go

```diff
@@ -92,11 +93,14 @@ func TextMessage(ctx context.Context, theURL, threadKey, text string) (MessageRe
 	}
 
 	client := &http.Client{}
+	start := time.Now()
 	resp, err := client.Do(req)
```

Comment:
In https://pix4dbug.atlassian.net/browse/PCI-4096 we should implement retry here... but we first need to learn more about Gchat rate limit and its headers...

Only then we can also log rate limit info for Gchat... Its still valuable to have duration and usual HTTP request info..